### ISSUE TITLE
support postgresql

### ISSUE THREAD

**Issue by arankaren** [pluja/Parasitter#137]
Is there support for postgresql?. I've only seen MariaDB in documentation

**Comment by FireMasterK** [comment]
I actually tried to run yotter with postgres, and when I did so, I was getting some stackoverflow and timeout exceptions, I think this will require some work.

**Comment by pluja** [comment]
Yes, as @FireMasterK said this needs some work. I will not invest time on implementing this for now as we have other priorities. If you (or anyone) wants to do it, you can go ahead an make a PR!

**Comment by arankaren** [comment]
@pluja  a developer named heckyel [heckyel.i2p](http://heckyel.i2p) shared the command to make Yotter work with Posgresql.

### Configuration for Postgres

1. Login as postgres
`sudo su - postgres`

1. Create base
`createdb namebase`

1. Create User (place a password for our user)
`createuser -P username`

1. Inside the database
`psql -d namebase`

1. Give permissions to the created user
`GRANT ALL PRIVILEGES ON DATABASE namebase TO username;`

1. Exit
`\q`

### Configuration Yotter

#### Requirements

`$ pip install psycopg2-binary`

#### Edit `.env` file

`DATABASE_URL=postgresql+psycopg2://username:password@localhost:5432/namebase`

**Comment by pluja** [comment]
Awesome! I'll test it and add this to the self-hosting docs :100: 

@arankaren have you already tested this?

**Comment by arankaren** [comment]
> Awesome! I'll test it and add this to the self-hosting docs 💯
> 
> @arankaren have you already tested this?

Yes, it works perfectly on my instance
